PWGHF/HFL/Tasks/taskHFmuCharmBeautySep.cxx

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \Task to estimate HF->mu in the forward direction and use the DCA observable to separate b-> mu, c-> mu.
+/// \Command to run on terminal : o2-analysis-hfmu-charm-beauty-separation | o2-analysis-fwdtrackextension --aod-file AO2D.root
+/// \Author : Shreyasi Acharya <shreyasi.acharya@cern.ch>, LPC, France
```

Comment:
These don't look like Doxygen commands. Please the existing files.
And there is not need to put a command there. If you really want, you can just specify that this workflow requires `o2-analysis-fwdtrackextension` as a dependency.